postcommit.py

```diff
@@ -0,0 +1,642 @@
+                  	       'message': description, 'commit_type': commit_type, 'parents': [],
+                   	       'commit_date': commit_date, 'push_time': getCurrentTime()
+                   	      }  
+
+			#  diff_changes temporarly not going to be used *may* be useful in future (just add **self.getDiffContent(parent,node)** in diff_changes )
+            		for parent in parents:
+                		log['parents'].append({'changeset' :parent[:12], 'branch' : parent[:12],
+                                      		       'diff_changes' : ""})
+            		if len(parents) == 2:
+                		log = self.updateMergeDetails(log, parents, parents_short)
+                		log['commit_type'] = 2
+
+            		log['files'] = self.filestatus(node)
+	    
+	    		self.logs.append(log)
+
+#	***************************************************************************************************************
+#	getDiffContent :
+#
+#			Show changes between two changeset
+#
+#			@param : changeset1,changeset2
+#			
+#			@return : [{'changed_lines': [1,4,6], 'file': 'test'}]
+#       *************************************************************************************************************** 
+
+    	def getDiffContent(self, changeset1, changeset2):
+        	diff_command = "git diff -U0 " + changeset1 + " " + changeset2 
+        	result = status_check(diff_command, self.module_dir)
+        	diff_changes = self.parseDiffContent(list(result))
+        	return diff_changes
+
+
+#	***************************************************************************************************************
+#	getDiffLines :
+#
+#			Find number of lines newly added in file
+#
+#			@param : changes
+#
+#			@return : [1,2,4,5]
+#       ***************************************************************************************************************
+    	def getDiffLines(self,changes):
+        	difflines = []
+        	for val in changes:
+            		changed_line_values = str(val)
+
+            	if changed_line_values.find(',') > -1:
+                	start_linenumber = int(changed_line_values.split(',')[0])
+                	total_changed_lines = int(changed_line_values.split(',')[1])
+                	for line in range(start_linenumber, start_linenumber + total_changed_lines):
+                    		difflines.append(line)
+            	else:
+                	difflines.append(int(changed_line_values))
+
+        	return difflines
+
+#       ***************************************************************************************************************
+#	parseDiffContent :
+#
+#			Parse diff contents for two changesets
+#
+#			@param : diff
+#			
+#			@return : [{'changed_lines': [1,4,6], 'file': 'test'}]
+#       *************************************************************************************************************** 
+    	def parseDiffContent(self, diff):
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename method "parseDiffContent" to match the regular expression ^[a-z_][a-z0-9_]{2,}$. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=python%3AS100)